migtests/scripts/run-test.sh

```diff
@@ -65,17 +65,21 @@ main() {
 	tail -20 ${EXPORT_DIR}/reports/report.txt
 
 	step "Export data."
-	export_data
-	ls -l ${EXPORT_DIR}/data
+	{ #TRY
+		export_data
+	}|| { #CATCH
```

Comment:
No need of curly braces for just a single command.
```
export_data || {
  ...
  ...
}
```

Reply:
done